assets/src/infrastructure/ui/layout/modal/index.ts

```diff
@@ -0,0 +1,15 @@
+export { default as Modal } from './Modal';
+
+export { default as ModalBody } from './ModalBody';
+
+export { default as ModalCloseButton } from './ModalCloseButton';
+
+export { default as ModalContent } from './ModalContent';
+
+export { default as ModalFooter } from './ModalFooter';
+
+export { default as ModalHeader } from './ModalHeader';
+
+export { default as ModalOverlay } from './ModalOverlay';
```

Comment:
oh I thought these had been done in the previous PR... 

don't feel so bad for making you change things now 😆